src/main/java/it/uniroma3/mat/extendedset/intset/ConciseSet.java

```diff
@@ -390,7 +390,7 @@ private static int getFlippedBit(int word)
    */
   private static int getLiteralBitCount(int word)
   {
-    return BitCount.count(getLiteralBits(word));
+    return Integer.bitCount(getLiteralBits(word));

```

Comment:
`Integer.bitCount()` compiles into a single POPCNT instruction (1 cpu tick) on all modern CPUs. Non-intrincified version is much slower